### ISSUE TITLE
New unit request: Equivalent

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#250]
https://en.wikipedia.org/wiki/Equivalent_(chemistry) 

ucumCode: `eq`

**Comment by steveraysteveray** [comment]
This is an interesting case, since it is not possible to state a conversion multiplier into amount of substance. Am I reading the wikipedia article correctly in that it states a conversion for an ionic reaction, but this would not be true for an arbitrary chemical reaction with different stoichiometries?

**Comment by dr-shorthair** [comment]
the [relevant table in the UCUM spec](https://ucum.org/ucum.html#para-45) appears to have a conversion factor to mole of 1 but there is a big footnote (reproduced below). 
I don't really understand it at all, but it is clearly important in that domain. Perhaps we need to ask a biochemist. 

> The amount of electrolytes (including acids and bases) is often reported as equivalents instead of amount of substance. This habit originates in the measuring technique of titration. The Unified Code for Units of Measure does not endorse using equivalents. We rather recommend to calculate the proper amount of substance after titration, so that 1 eq of Na+ ions is 1 mol, but 1 eq of Ca++ ions is 0.5 mol. The problem with equivalents is that the measurement results are difficult to compare because their magnitude depends on the degree of ionization of the substance. That is to say, the meaning of equivalents depend not only on the substance, but also on the state that the substance is in. For example, in iron we have to distinguish Fe2+ from Fe3+, so that noone can be sure how much 1 eq of iron really is.

**Comment by steveraysteveray** [comment]
Indeed. And this idea doesn't seem to generalize to, say, organic chemistry. Just ionic compounds (I think).
...but I'm no biochemist.

**Comment by dr-shorthair** [comment]
Raised this with one of the vice-presidents of IUPAC today. 
He says he hasn't come across Equivalents since he was at school ... 
Nevertheless, since it is used in some active domains, I guess we need at least a placeholder for it.

**Comment by steveraysteveray** [comment]
Is it still? I suppose we could put a zero multiplier to guard against someone getting an incorrect answer. I'm still not completely convinced it belongs in QUDT though. Interested to hear other's thoughts on this.

**Comment by dr-shorthair** [comment]
These from a vocabulary that is actively used by the global oceanography community, 

http://vocab.nerc.ac.uk/collection/P06/current/EQPL/ 
http://vocab.nerc.ac.uk/collection/P06/current/UEQL/ 
http://vocab.nerc.ac.uk/collection/P06/current/MEKG/
http://vocab.nerc.ac.uk/collection/P06/current/MEQL/ 

as indicated by their appearance in the SeaDataNet service
https://bit.ly/3d9pPsL

**Comment by steveraysteveray** [comment]
OK you got me there. Convinced me that we should add it. ￼￼

**Comment by meganrwong** [comment]
Hello. Thanks for getting this up Simon, and for looking into Steve. I'm not a chemist either, the vice-president of IUPAC certainly would know better!

We broker data that comes from a number of soil laboratories in Australia. Some of this is historical data, but we are still getting data in in equivalents. 

the data comes in from labs labelled with units 'meq/100g' . It is a unit associated with the measure of concentration of cations in the soil.

I think - 
MilliEquivalentPer100g = MilliEquivalentPerHectoGram = meq% = cmol/kg = cmolc/kg

Since we were after applying UoM that were available as persistent identifiers, we (for now) used -
http://registry.it.csiro.au/def/environment/unit/MilliEquivalentPerHectoGram

The notes here say it is, or should be, a redundant term and we should use 
cmole(+)/kg
I don't think there is a cmol/kg UoM in this QUDT version?

It seems we could also covert to mg/kg (which is equiv to ppm also, which I note you have added in this version which is good. It is commonly used in soil chem and don't think was in the last version). But, I think suspect soil data providers will like to see their results in units they are familiar with ........

**Comment by meganrwong** [comment]
oh sorry, here is the link, 
http://www.backpaddock.com.au/news/soilmate-changes-reporting-units-for-cations/

**Comment by meganrwong** [comment]
Do we perhaps then also need a cmole(+)/kg, I wonder....
